WebService/v1/Controllers/StatusController.cs

```diff
@@ -109,8 +113,10 @@ public async Task<StatusApiModel> GetAsync()
             }
             catch (Exception e)
             {
-                errors.Add("Unable to fetch simulation status");
-                this.log.Error("Unable to fetch simulation status", e);
+                var msg = "Unable to fetch simulation status";
```

Comment:
we're removing information about the simulation from this endpoint, so we might not need this. Could you double check with @hathind-ms ?

Reply:
don't need to log here.

Reply:
done